corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -295,7 +295,7 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
 <script type="text/html" id="repeat-juncture-fullform-ko-template">
   <div class="panel panel-default rep">
     <div class="panel-heading">
-      <h3 class="caption" data-bind="html: header" tabindex="0"></h3>
+      <h3 class="caption" data-bind="html: caption()" tabindex="0"></h3>
```

Comment:
This change makes template for repeat use caption instead of header, just like all other templates using caption/caption_markdown.
1. I am not aware of formplayer internal workings but I could confirm that formplayer sends the same value of caption and header for repeater in the response from formplayer.
2. We don't support formatting for repeat group label, so it will never have caption_markdown. So an alternate could have been to just use `text: header` but it made sense to instead use caption since other templates use that as well.